includes/css/scss/_module-header-brand-dealership.scss

```diff
@@ -73,11 +52,6 @@
 	align-items: center;
 	text-decoration: none;
 	margin-right: $gap-size;
```

Comment:
1. Please change the `$gap-size` value to match the [Volkswagen factory site](https://www.volkswagen.com.au/en.html). There's is smaller.
2. Please change your variable name `$gap-size` to something more specific. It's unclear what it's for in `_variables.scss` but it's very specific for one use and can't be repurposed anywhere else on the site. `vw-logo-gap` is better, but feel free to create your own. Please add a short comment above the variable in `_variables.scss` explaining it's purpose.

Reply:
Resolved in 946a44b and ae05245